spec/system/planning_applications/policy_assessment_spec.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Planning Application Assessment", type: :system do
+  let!(:assessor) { create :user, :assessor, local_authority: @default_local_authority }
```

Comment:
I've checked this works equivalently for the reviewer (Manager) role. Not sure if we need to add some extra specs to validate this or whether it's just going to be annoying code duplication